api/agent_config.go

```diff
@@ -0,0 +1,43 @@
+package api
+
+type K3sAgentConfig struct {
+	Config string `json:"config,omitempty" yaml:"config,omitempty"`
```

Comment:
Is there any properties valid for agent but not for the server?

Reply:
don't know @jzhoucliqr. @paynejacob may know more.